modules/core/src/test/java/org/apache/ignite/testframework/ListeningTestLogger.java

```diff
@@ -29,11 +30,6 @@
  * It can be useful in tests to ensure that a specific message was (or was not) printed to the log.
  */
 public class ListeningTestLogger implements IgniteLogger {
```

Comment:
I think we should remove constructors where "debug" is used, so as not to confuse users, let it remain at discretion of core team.